content/en/docs/helm/helm.md

```diff
@@ -89,6 +89,8 @@ By default, the default directories depend on the Operating System. The defaults
 * [helm package](helm_package.md)	 - package a chart directory into a chart archive
 * [helm plugin](helm_plugin.md)	 - install, list, or uninstall Helm plugins
 * [helm pull](helm_pull.md)	 - download a chart from a repository and (optionally) unpack it in local directory
+* [helm push](helm_push.md)	 - push a chart to remote
+* [helm registry](helm_registry.md)	 - login to or logout from a registry
```

Comment:
I think the build is breaking due to not finding these two files?

Reply:
missed a git add. Thanks.